packages/blade/src/components/Accordion/AccordionItem.tsx

```diff
@@ -38,6 +38,20 @@ type AccordionItemProps = {
   _index?: number;
 } & TestID;
 
+/**
+ * On React Native if the `AccordionItem` has a lengthy description which renders a `Text` spanning multiple lines,
+ * it sometimes messes up the layout calculation (it thinks of multiline as a single line before flowing in the UI).
+ * And during the expanding / collapsing animation, text reflows causing words to jump around across lines.
+ *
+ * Rendering a blank `Text` at the end seems to fix all this 🤯
+ */
+const reactNativeMultilineTextOverflowFix = (
+  // make this hidden from screen readers
+  <BaseBox {...makeAccessible({ hidden: true })}>
+    <Text> </Text>
```

Comment:
Prettier could remove this space at some point. Faced such an issue a while back, better to be explicit.
```suggestion
    <Text>{' '}</Text>
```